### PR TITLE
[qacode] fix for issue #175

### PR DESCRIPTION
**Pull request by netzulo**
# Description

- Fixing issue from CI, codeclimate : https://codeclimate.com/github/netzulo/qacode/qacode/core/webs/css_properties.py#issue_5b1d7a131112470001000083

## Issues related

- [x] #175 
- [x] #174 
- [x] #177

### CODE REVIEW

File: qacode/core/enums/enum_base.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from enum import Enum
+
+
+class EnumBase(Enum):
+    """Enum Class with base methods to be used
+        at inherit classes
+    """
+
+    @classmethod
+    def get_properties(cls):
```
Comment (Terseus) [inline qacode/core/enums/enum_base.py]: Not needed, it's already implemented in `Enum`: https://docs.python.org/3/library/enum.html#iteration

File: qacode/core/enums/enum_base.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        """Return enum values"""
+        return [item.value for item in type(cls)]
+
+    @classmethod
```
Comment (Terseus) [inline qacode/core/enums/enum_base.py]: While not really implemented, it's more idiomatic and fastest to try to get the enum from the constructor class using the value:

```python
from enum import Enum

class MyEnum(Enum):
    A = 1
    B = 2

MyEnum(1)  # Returns MyEnum.A
MyEnum(3)  # Raises ValueError
```

At a minimum, I would rewrite the `has_property` to use the class constructor instead to iterate over the values.

File: qacode/core/enums/enum_base.py (hunk 1 above)
Comment (netzulo) [reply qacode/core/enums/enum_base.py]: reading about this... 🤔

File: qacode/core/enums/enum_base.py (hunk 2 above)
Comment (netzulo) [reply qacode/core/enums/enum_base.py]: will be resolved at #181

File: qacode/core/enums/enum_base.py (hunk 1 above)
Comment (netzulo) [reply qacode/core/enums/enum_base.py]: will be resolved at #181

**Pull request merged**